giskard_vision/image_classification/types.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Tuple
+
+import numpy as np
+
+from giskard_vision.core.types import (
+    IMAGE_TYPE,
+    META_TYPE,
+    PredictionResultBase,
+    TypesBase,
+)
+
+CLASSIFICATION_LABEL_TYPE = np.ndarray  # Probabilities for each class
```

Comment:
The classification type is not probabilities.

We need to implement a threshold for binary classification models to get the predicted classes
We already have in place the argmax for the multi-classifcation case

In both cases though, I would choose between:
- `int`: label_id
- `string`: label 

I'm more in favour of label as it'll be more readable when we use `model.predict`

Reply:
I assumed an N-dimensional array to have both multiple labels for one prediction and multiple predictions.